go/cmd/vtctldclient/command/root.go

```diff
@@ -36,8 +41,9 @@ var (
 	// VtctldClientProtocol is the protocol to use when creating the vtctldclient.VtctldClient.
 	VtctldClientProtocol = "grpc"
 
-	client        vtctldclient.VtctldClient
-	traceCloser   io.Closer
+	client      vtctldclient.VtctldClient
+	traceCloser io.Closer
+
```

Comment:
this change seems unnecessary?

Reply:
Just an additional whitespace similar to the one below commandCancel to keep the related variables separate I suppose. We can revert in a following PR if it is felt to be unnecessary.